notebooks/ArynPartitionerPython.ipynb

```diff
@@ -81,7 +83,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "aryn_api_key = 'YOUR-ARYN-API-KEY'"
+    "from sycamore.utils.aryn_config import ArynConfig\n",
+    "aryn_api_key = ArynConfig.get_aryn_api_key()\n",
+    "\n",
+    "# you can also read directly from env variable like this\n",
```

Comment:
We shouldn't need this; our APIs  automatically read the api keys. 
The main reason you need it is if you stored the key somewhere else, in which case `ArynConfig.get_aryn_api_key("/home/eric/keys/aryn/config.yaml")` would work.

Reply:
thanks, will update the notebooks.